ast/dml.go

```diff
@@ -399,58 +568,68 @@ const (
 type InsertStmt struct {
 	dmlNode
 
+	Replace     bool
```

Comment:
I think insert and replace are different statement, so maybe it is not a good name.